tools/create_certs/root.cert

```diff
@@ -0,0 +1,28 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Which root is this?

Reply:
It was used when performing JITP manually, but is unnecessary now. Just deleted :)

Reply:
It looks like this root certificate file is still being added as part of your PR. Do you still have a pending commit that removes it?

Reply:
That's my bad. I had incorrectly removed the file from remote and not local, and then force pushed my branch. Just updated. Thanks for pointing that out.